### ISSUE TITLE
[JENKINS-72822] Remove class loading workarounds

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jakarta-activation-api-plugin#105]
<p>See <a href="https://github.com/jakartaee/jaf-api/pull/145#issuecomment-1981960181;" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/pull/145#issuecomment-1981960181;</a> as of <a href="https://github.com/jakartaee/jaf-api/pull/145" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/pull/145</a> the workaround in <a href="https://github.com/jenkinsci/jakarta-mail-api-plugin/tree/7e494506cd6dc23f8144afbed256a44126bd666c/src/main" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jakarta-mail-api-plugin/tree/7e494506cd6dc23f8144afbed256a44126bd666c/src/main</a> can be removed, but only after the Jakarta Activation API plugin and Jakarta Mail API plugin are merged into a single plugin.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=basil">basil</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-72822" target="_blank">Remove class loading workarounds</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>component(s)</b>: jakarta-activation-api-plugin, jakarta-mail-api-plugin
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 20260107-133739
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>See <a href="https://github.com/jakartaee/jaf-api/pull/145#issuecomment-1981960181;" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/pull/145#issuecomment-1981960181;</a> as of <a href="https://github.com/jakartaee/jaf-api/pull/145" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/pull/145</a> the workaround in <a href="https://github.com/jenkinsci/jakarta-mail-api-plugin/tree/7e494506cd6dc23f8144afbed256a44126bd666c/src/main" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jakarta-mail-api-plugin/tree/7e494506cd6dc23f8144afbed256a44126bd666c/src/main</a> can be removed, but only after the Jakarta Activation API plugin and Jakarta Mail API plugin are merged into a single plugin.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>basil:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-72822?focusedId=445991&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-445991">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hoping for an upstream fix in <a href="https://github.com/jakartaee/jaf-api/issues/161" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/issues/161</a>.</p>
</pre>
</details>

</ul>
</details>

<p>Hoping for an upstream fix in <a href="https://github.com/jakartaee/jaf-api/issues/161" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jakartaee/jaf-api/issues/161</a>.</p>
